lib/clerk/rack_middleware_v2.rb

```diff
@@ -162,10 +166,15 @@ def call(env)
         return signed_out(env)
       end
 
-      token = verify_token(cookie_token)
+      begin
+        token = verify_token(cookie_token)
+        return signed_out(env) unless token
 
-      if token && token["iat"] && client_uat && Integer(client_uat) <= token["iat"]
-        return signed_in(env, token, cookie_token)
+        if token["iat"] && client_uat && Integer(client_uat) <= token["iat"]
+          return signed_in(env, token, cookie_token)
+        end
+      rescue JWT::ExpiredSignature, JWT::InvalidIatError
```

Comment:
Ditto: shouldn't this be only wrapping `verify_token()`?

Reply:
same reason as https://github.com/clerkinc/clerk-sdk-ruby/pull/30#discussion_r1127968320